src/amber/cli/commands/database.cr

```diff
@@ -55,7 +55,7 @@ module Amber::CLI
             begin
               Micrate::Cli.run_up
             rescue e : IndexError
-              exit! "no migrations to run"
+              exit! "No migrations to run in #{MIGRATIONS_DIR}."
```

Comment:
The reason behind writing the error in lower case was to match the output given from Micrate. The error messages returned by Micreate are all lowercase.